### PR TITLE
remove readAt and expireTimestamp from cache objects

### LINKED ISSUE #2540: [Bug]: Cache inconsistency
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

If two Actions are working with the same value in the cache at the same time, it's possible to get some inconsistent behavior. I was load testing a new application I wrote with Actionhero and was able to reproduce this fairly regularly. 

Example:
* The cache holds the value `0` at `key`
* Action A saves `1` to `key`
* At almost exactly the same time, Action B loads `key` from the cache
* B gets `0` back just before A saves `1`
* A second later, loading `key` from the cache returns `0`

### Expected Behavior

I believe this line should be removed: https://github.com/actionhero/actionhero/blob/main/src/modules/cache.ts#L219. With that in place, it's possible for a call to `cache.save` to resolve and then have the value it just saved overwritten immediately by another call to `cache.load` on the same node. 

I'd be happy to open a PR for this, if that helps. I tried to join the Actionhero Slack, but https://slack.actionherojs.com/ is not responding.

### Stack Trace

_No response_

### Environment

OSX
Actionhero version 28.3.3
Node.js version 18.16.0

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by rogorman9**
Fixes #2540

### CODE REVIEW

File: src/modules/cache.ts [hunk 1]
```diff
@@ -191,48 +180,21 @@ export namespace cache {
...
-    }
-
     lockOk = await cache.checkLock(key, options.retry);
     if (lockOk !== true) throw new Error(CacheErrorMessages.locked);
 
-    await client().set(redisPrefix + key, JSON.stringify(cacheObj));
-    if (expireTimeSeconds) {
-      await client().expire(redisPrefix + key, expireTimeSeconds);
+    if (options.expireTimeMS) {
+      await client().pexpire(redisPrefix + key, options.expireTimeMS);
```
Comment (evantahler) [inline src/modules/cache.ts:187]: oh yay, we got an expire in ms method!

**Review by evantahler — COMMENTED** [review]
This looks good!
As none of the method signatures changed, but the body content of the cached objects did... do you think this should be a breaking change or not 🤔 ?

**Comment by evantahler** [comment]
cc @gcoonrod - what are you thoughts on if this would be a breaking change or not?

**Comment by gcoonrod** [comment]
> cc @gcoonrod - what are you thoughts on if this would be a breaking change or not?

Yes, it's a change to the members of the pubic `CacheObject` API, which I can imagine some projects might have used.

**Review by gcoonrod — APPROVED** [review]

**Comment by evantahler** [comment]
Related PRs for the version bump https://github.com/actionhero/actionhero/pull/2542 and docs update https://github.com/actionhero/www.actionherojs.com/pull/456

**Pull request merged**